New Prosec stamp and minor Justice loadout adjustments

Tomce795 opened the pull request:
## About the PR
Added new stamps for Prosecutor. Also adjusted their round-spawn equipment so they start with the stamp and with a Space Law book (also added one for Clerk),

## Why / Balance
There are specific documents that can only be approved by certain members of Justice. Prosecutor can approve things like warrants and they typically write out paperwork for prosecuting during a trial, But Prosec only has access to the Lawyer stamp, like Attorney. This should help differentiate who actually stamped what and miiiight encourage more Prosec to do more paperwork.

## Technical details
Added a Prosecutor stamp, stamp component and appropriate prototype.
Adjusted Prosecutor round start equipment to spawn with the stamp.
Added a Space Law book to both Prosecutor and Clerk on round start.

## Media
![Stamp example](https://github.com/user-attachments/assets/2365706d-99bd-4fde-87bf-721df5d8765e)

## Requirements
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
Probably none?

**Changelog**
:cl:
- add: Added a new stamp for Prosecutor and added it to their starting loadout. Also added the Space Law book to the starting loadout for both Prosec and Clerk.

Tomce795 committed:
Prosec stamp art

Tomce795 committed:
Clerk stamp art

Tomce795 committed:
Prosec and Clerk stamp stuff

Tomce795 committed:
respective stamps and space law book to roundstart

Tomce795 committed:
Merge branch 'justice-stamps' of https://github.com/Tomce795/Delta-v into justice-stamps

Tomce795 commented:
Unironically unsure if the new sprites are supposed to be in here or the Delta-V folder which has the mantis one. I'm sure one of you's will yell at me thou :blunt:

Avalon-Proto commented:
I feel like I should mention this. The clerk already has a stamp which is called the Notary Stamp. So the clerk stamp here is irrelevant

Tomce795 commented:
Guh. Where does this stamp spawn? In a Clerk locker or something?

Avalon-Proto commented:
Yep, it does

Tomce795 commented:
Okay general idea is to then to remove the Clerk stamp, make the prosec one the only one added. Also probably shift about the stamp to the Delta-V resource folder (but the notary stamped paper sprite isn't there for some reason?). Do Prosec get their own locker? If not, I'll keep the loadout changes so that they spawn with it.

Radezolid commented:
Prosec doesn't get a locker, gotta add it to their loadout.

Tomce795 committed:
Merge branch 'DeltaV-Station:master' into justice-stamps

Tomce795 committed:
Prosec stamp stuff

Tomce795 committed:
Reverted accidentally added Clerk stamp

Tomce795 committed:
Merge branch 'justice-stamps' of https://github.com/Tomce795/Delta-v into justice-stamps

Tomce795 committed:
Imagine forgetting to press save couldn't be me

Something something removed the edits to upstream localization, moved to the delta v one.

Tomce795 committed:
FORGOR

removed the stamps from Upstream Resource folder

Tomce795 committed:
minor meta.json fixes

Tomce795 commented:
I dunno why the upstream files are still... modified, but that's to figure out later or to webedit away :blunt:

I believe the other stuff is okay, so nap time for me.

Unkn0wnGh0st333 commented:
Uhh change the 🆑 to only be one bullet, don't need two different change logs for the same PR (unless it's a massive PR) :blunt:

MilonPL commented on Resources/Locale/en-US/paper/stamp-component.ftl:
untouch this file

MilonPL commented on Resources/Prototypes/Entities/Objects/Misc/rubber_stamp.yml:
untouch

Tomce795 committed:
Untouched upstream files

Tomce795 committed:
Merge branch 'master' into justice-stamps

Tomce795 committed:
Merge branch 'justice-stamps' of https://github.com/Tomce795/Delta-v into justice-stamps

Tomce795 commented:
Untouched upstream files. `git checkout upstream/master` to the rescue. GitHub is wacky and confuzzling but I think this is now good to merge after tests.

Tomce795 committed:
Merge branch 'master' into justice-stamps

Tomce795 committed:
Merge branch 'master' into justice-stamps

Tomce795 committed:
Merge branch 'master' into justice-stamps

Tomce795 committed:
Merge branch 'master' into justice-stamps
